Keas.Mvc/Controllers/PersonAdminController.cs

```diff
@@ -190,10 +191,12 @@ public async Task<IActionResult> UploadPeople(PeopleImportModel model)
             var userName = User.GetNameClaim();
             var team = await _context.Teams.FirstAsync(t => t.Slug == Team);
             using (var reader = new StreamReader(model.File.OpenReadStream()))
-            using (var csv = new CsvReader(reader))
+            using (var csv = new CsvReader(reader, new CsvConfiguration(CultureInfo.InvariantCulture)
```

Comment:
@sprucely Not a big deal, but I'd put the "new CsvCo..." on a new line and indent.
Took me a minute to see that the using didn't get closed out.